### ISSUE TITLE
warning about urllib or something

### ISSUE THREAD

**Issue by dmd** [bbfrederick/rapidtide#95]
(I have only tested this through the Docker container.)

Observe:

```
$ docker run -it fredericklab/rapidtide rapidtide
/usr/local/miniconda/lib/python3.9/site-packages/requests/__init__.py:102: RequestsDependencyWarning: urllib3 (1.26.8) or chardet (5.0.0)/charset_normalizer (2.0.4) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({})/charset_normalizer ({}) doesn't match a supported "
mklexists: False
usage: rapidtide
       [-h]
etc
etc
etc
```

**Comment by dmd** [comment]
Your "fix" had no effect.

**Comment by dmd** [comment]
Fixed in 20216895605735a37cf3454a550d4b74b756e8b4.
